SPEC-annex-B-elf-embedding.md

```diff
@@ -0,0 +1,29 @@
+# Annex B
+
+Annex B contains a method of embedding OmniBOR ID into elf files.
```

Comment:
I think "elf files" should be written as "ELF files," since "ELF" is an acronym.

Reply:
Fixed